h/static/styles/annotations.scss

```diff
@@ -178,12 +185,13 @@ privacy {
       margin: 0;
     }
 
-    .annotation-body, .annotation-footer {
+    .annotation-body, .annotation-footer, .tags-read-only {
```

Comment:
Is this change necessary? I think the right change here is to make that an `annotation-body` instead of an `annotation-section`. I made the other tags section into "a body" recently, but I missed the read-only one.